.changeset/long-spoons-pump.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap/guided-answers-extension-types': minor
```

Comment:
@unseen1980, in order to generate a new version of the extension itself, we need also to create a changeset for `sap-guided-answers-extension`, even if it is not changed, the version needs to be bumped.